README.md

```diff
@@ -134,6 +134,9 @@ clean out any 'bad' local git repo you already have.
 --create-debug-image
 create a debug-image archive with the debug symbols.
 
+--create-jre-image
```

Comment:
Please also add this new option to the man page here: https://github.com/adoptium/temurin-build/blob/master/makejdk-any-platform.1

Reply:
Option added, please see makejdk-any-platform.1below.